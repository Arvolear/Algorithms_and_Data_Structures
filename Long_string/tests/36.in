30218155887177905561802061036581703738228957
674165169130288960695325743711670086705638978766