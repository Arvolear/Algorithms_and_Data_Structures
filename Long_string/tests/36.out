674195387286176138600887545772706668409377207723
674134950974401782789763941650633505001900749809
20372028174484729760808074789747984719560074209760778468674795124039487109244547810969327062
22309